CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+#### 1.0.1 (2020-05-11)
```

Comment:
@apetro Kesong used a package to generate the changelog, should he put it in to the dependency?

Reply:
Thanks for the answer! I added one line in the readme file saying we are using a package to generate the changelog and did not put the package itself in our dependency.